app/javascript/mastodon/components/status_content.js

```diff
@@ -46,10 +51,15 @@ export default class StatusContent extends React.PureComponent {
     }
   }
 
+  componentDidMount () {
+    this._updateStatusLinks();
+  }
+
   componentDidUpdate () {
     if (this.props.onHeightUpdate) {
       this.props.onHeightUpdate();
     }
+    this._updateStatusLinks();
```

Comment:
@Gargron However, we set event listeners on `Permalink` components now. If we're going to ignore lost of listeners on `Permalink`, we shouldn't set event listeners on `Permalink` any time, for consistency.